frontend/packages/client/src/components/Community/CommunityEditorLinks.js

```diff
@@ -129,11 +128,12 @@ export default function CommunityEditorLinks(props = {}) {
     setIsUpdating(false);
   };
 
-  const { isValid } = useLinkValidator({ links, initialValues: props });
+  const { isValid, hasChangedFromOriginal } = useLinkValidator({
+    links,
+    initialValues: fields,
+  });
 
-  useEffect(() => {
-    setEnableSave(isValid);
-  }, [isValid]);
```

Comment:
this wasn't necessary since a hook is re-rendering the object when fields are valid or are different from originals